bin/submarine.sh

```diff
@@ -55,4 +55,4 @@ if [[ ! -d "${SUBMARINE_LOG_DIR}" ]]; then
   $(mkdir -p "${SUBMARINE_LOG_DIR}")
 fi
 
-exec $JAVA_RUNNER $JAVA_OPTS -cp ${SUBMARINE_SERVER_CLASSPATH} ${SUBMARINE_SERVER_MAIN} "$@" >> "${SUBMARINE_SERVER_LOGFILE}" 2>&1
+exec $JAVA_RUNNER $JAVA_OPTS -cp ${SUBMARINE_SERVER_CLASSPATH} ${SUBMARINE_SERVER_MAIN} "$@" | tee -a "${SUBMARINE_SERVER_LOGFILE}" 2>&1
```

Comment:
Use `tee` copies the data to `STDOUT` ? Is it the really reason caused the the logs empty?

Reply:
'kubectl logs' shows the stdout of the container in the pod. 
The cause of 'kubectl logs' to be empty is that the container only direct the output to the log file. However, if we want to show the output by 'kubectl logs', we should direct it to stdout too.

Reply:
LGTM. Thanks for your contribution. Will merge if no more comments.